tests/analyze.test/t15.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+${CDB2SQL_EXE} ${CDB2_OPTIONS} ${DBNAME} default "CREATE TABLE t(a int) partitioned by manual retention 2"
+
+SHARDNAME=$(${CDB2SQL_EXE} -tabs ${CDB2_OPTIONS} ${DBNAME} default "select shardname from comdb2_timepartshards where name='t' limit 1")
+
+${CDB2SQL_EXE} ${CDB2_OPTIONS} ${DBNAME} default "ANALYZE \"$SHARDNAME\""
```

Comment:
For the example above, can you make this work? `ANALYZE t`

Reply:
Done. I'll update the test to verify that "Analyze t" works